contrib/k8s/README.md

```diff
@@ -27,17 +26,14 @@ dex needs a secret key for encrypting private keys in the database. These can be
 
 [k8s-secrets]: http://kubernetes.io/v1.0/docs/user-guide/secrets.html
 
-```
-kubectl create -f dex-secrets.yaml
-```
+The secret will be created as part of the next command, from `dex-overlord.yaml`
```

Comment:
Be careful using `base64` in examples, it's usage varies based on darwin and linux.